src/main/java/com/google/sps/model/GmailClient.java

```diff
@@ -160,9 +144,28 @@ static String fromEmailQuery(String from) {
    *
    * @param isImportant true if filtering for important emails, false otherwise
    * @return string to use in gmail (either client or APi) to find emails that match these criteria
-   *     Trailing space added to string so multiple queries can be concatenated
    */
   static String isImportantQuery(boolean isImportant) {
     return isImportant ? "is:important" : "";
   }
+
+  /**
+   * Creates Gmail query to find emails with the passed word in the subject line
+   *
+   * @param words words to check subject line for.
+   * @return query string to use in gmail to find emails that contain the words
+   */
+  static String wordsInSubjectLineQuery(List<String> words) {
```

Comment:
Make it clear this query will match any of the words. I suggest changing the method name and updating the documentation. 

(Also, using the term "words" is a little too specific. You don't do anything with spaces or other word delimiters. Perhaps a list of "phrases" or "tokens" would be more general.)

Reply:
Agreed - fixed!